### ISSUE TITLE
COMPILE-FILE-ERROR

### ISSUE THREAD

**Issue by smvamse** [40ants/cl-flamegraph#6]
When I `(ql:quickload :flamegraph)`, I get the following error:
![Screenshot 2021-06-15 at 6 29 24 AM](https://user-images.githubusercontent.com/84125167/121977255-33f52800-cda3-11eb-9ac5-b09e7ca1a564.png)

**Comment by svetlyak40wt** [comment]
Yeah, I already know. SBCL 2.1.2 dropped `map-trace-samples` function.

I tried to find a solution once, but without any luck. Still have now time to dive into the problem again :(

See also related issue https://github.com/40ants/cl-flamegraph/issues/3

**Comment by jmercouris** [comment]
@stassats any alternate solution?
